test/caeser_test.rb

```diff
@@ -0,0 +1,65 @@
+# YOUR TESTS GOES HERE
+# I PITY THE FOOL WHO DOESNT WRITE TESTS
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/caeser'
+
+class CaeserTest < MiniTest::Test
```

Comment:
Your tests look great! I like how you started simple and moved to complex cases. I know exactly what your code does just by looking at the tests.